src/geonovum/leafletfigures.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Module: geonovum/leafletfigures
+ * Makes figures scalable via zoom and pan function
+ */
+import L from "geonovum/deps/leaflet";
+import easyButton from "geonovum/deps/easy-button";
+
+export function run(conf, doc, cb) {
+  doc.respecIsReady.then(function() {
+    Array.from(doc.querySelectorAll("figure.scalable img")).forEach(function(
+      image
+    ) {
+      var container = image.parentElement;
+      var div = doc.createElement("div");
+      div.id = "leaflet-figure";
+      container.replaceChild(div, image);
+      var map = L.map(div, {
+        maxZoom: 4,
+        minZoom: -4,
+        center: [0, 0],
+        crs: L.CRS.Simple,
+      }).setView([image.height / 2, image.width / 2], 1);
+      var imageBounds = [[0, 0], [image.height, image.width]];
+      L.easyButton("fa-arrows-alt", function() {
+        window.open(image.src, "_blank");
+      }).addTo(map);
+      L.easyButton("fa-globe", function() {
+        map.fitBounds(imageBounds);
+      }).addTo(map);
+      L.imageOverlay(image.src, imageBounds).addTo(map);
+      map.fitBounds(imageBounds);
+    });
+  });
+  cb();
+}
```

Comment:
If I change the import it works:
```javascript
import * as L from "geonovum/deps/leaflet";
import * as easyButton from "geonovum/deps/easy-button";
```

The page, locally generated, is slow. The generator plugin is not loading:
```
Plugin  took too long. Object { run: run(), 1 meer… }  base-runner.js:51:11
	toRunnable/</</timerId< http://localhost:8080/js/core/base-runner.js:51:11
Error: Plugin  took too long.
Stacktrace:
toRunnable/</</timerId<@http://localhost:8080/js/core/base-runner.js:52:18
  base-runner.js:105:15
	_callee$ http://localhost:8080/js/core/base-runner.js:105:15
	tryCatch http://localhost:8080/js/deps/regenerator.js:64:37
	invoke http://localhost:8080/js/deps/regenerator.js:299:22
	defineIteratorMethods/</prototype[method] http://localhost:8080/js/deps/regenerator.js:116:16
	step http://localhost:8080/js/core/base-runner.js:15:24
	step/< http://localhost:8080/js/core/base-runner.js:28:15
```